delta/plugins/elasticsearch/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/client/ElasticSearchClientSpec.scala

```diff
@@ -224,5 +225,35 @@ class ElasticSearchClientSpec(override val docker: ElasticSearchDocker)
           jsonContentOf("elasticsearch-results.json", "index" -> index)
       }
     }
+
+    "delete documents by" in {
+      val index = IndexLabel(genString()).rightValue
+
+      val operations = List(
+        ElasticSearchBulk.Index(index, "1", json"""{ "field1" : 1 }"""),
+        ElasticSearchBulk.Create(index, "2", json"""{ "field1" : 3 }"""),
+        ElasticSearchBulk.Update(index, "1", json"""{ "doc" : {"field2" : "value2"} }""")
+      )
+
+      {
+        for {
+          // Indexing and checking count
+          _        <- esClient.bulk(operations)
+          _        <- esClient.refresh(index)
+          original <- esClient.count(index.value)
+          _         = original shouldEqual 2L
```

Comment:
It could be nice to say `givenDocumentsExist` or similar and hide this setup code

Reply:
I don't want to spend too much time on it as the ES client is likely to change a lot when the` akka http client` it relies on will be kicked out.